# Enum: encoder type
uint8 ENCODER_TYPE_ABSOLUTE = 0
uint8 ENCODER_TYPE_INCREMENTAL = 1

float32 omega_left
float32 omega_right
int32 left_ticks
int32 right_ticks